docs/cardanoNuFiConnect.md

```diff
@@ -0,0 +1,222 @@
+# NuFiConnect (Cardano)
+
```

Comment:
I'd add here a paragraph clarifying what is this for as it may not be obvious at first sight:

```
This library integrates your DApp with the NuFi web wallet injected into it is as a widget. Users can seamlessly onboard/log in into your DApp using social providers such as Google or Discord while being able to manage their wallet in the NuFi interface.
```